src/main/java/net/skinsrestorer/shared/storage/SkinStorage.java

```diff
@@ -320,11 +324,16 @@ public Object getSkinData(String name, boolean updateOutdated) {
     }
 
     public Object getSkinData(String name) {
-        return this.getSkinData(name, true);
+        return this.getSkinData(name, false);
     }
 
     private boolean isOld(long timestamp) {
-        return timestamp + TimeUnit.MINUTES.toMillis(Config.SKIN_EXPIRES_AFTER) <= System.currentTimeMillis();
+    	// Check so that updating can be disabled by setting expiration to zero
+    	if (Config.SKIN_EXPIRES_AFTER == 0 ) {
```

Comment:
having SKIN_EXPIRES_AFTER on 0 to disable automatic skin update could be misinterpreted fast.
Especially for us having a wide verity of langue speakers.

I think adding another option to disable "automatic update skins"  would be a better option to have it disabled and have 0 always return true

Reply:
SkinAutoUpdates added to config.yml
SKIN_AUTO_UPDATES added to Config.java
isOld method in SkinStorage.java updated to use SKIN_AUTO_UPDATES